packages/application-shell/src/components/navbar/main-navbar.styles.ts

```diff
@@ -85,7 +86,10 @@ const leftNavigationOpenStyles = css`
   .body__menu-open ${MenuListItem} {
     height: auto;
     min-height: ${NAVBAR.itemSize};
-    width: 100%;
+    width: calc(
+      ${NAVBAR.sublistIndentationWhenExpanded} - 2 *
+        ${uiKitDesignTokens.spacing25}
+    );
```

Comment:
I think it's now ready for review (and it has been re-deployed)
The main change lies in using fixed values for the menu list item's width, replacing the previous reliance on 100% width and hover-applied padding (to address any cross-browser inconsistencies)